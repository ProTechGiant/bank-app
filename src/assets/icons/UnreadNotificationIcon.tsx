import Svg, { Circle } from "react-native-svg";

import { IconProps } from "@/assets/icons";

export function UnreadNotificationIcon({ width = 6, height = 6, color = "#FF371E" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 6 6" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Circle cx="3" cy="3" r="3" fill={color} />
    </Svg>
  );
}
