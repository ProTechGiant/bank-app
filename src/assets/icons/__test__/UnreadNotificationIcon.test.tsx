import { create } from "react-test-renderer";

import { UnreadNotificationIcon } from "../UnreadNotificationIcon";

describe("icon/UnreadNotificationIcon", () => {
  it("renders a UnreadNotificationIcon", () => {
    const result = create(<UnreadNotificationIcon />);

    expect(result).toMatchSnapshot();
  });
});
