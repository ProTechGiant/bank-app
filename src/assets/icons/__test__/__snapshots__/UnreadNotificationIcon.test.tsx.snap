// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/UnreadNotificationIcon renders a UnreadNotificationIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="6"
  bbWidth="6"
  fill="none"
  focusable={false}
  height={6}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 6,
        "width": 6,
      },
    ]
  }
  vbHeight={6}
  vbWidth={6}
  width={6}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGCircle
      cx="3"
      cy="3"
      fill={
        {
          "payload": 4294915870,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
      r="3"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
